src/action.js

```diff
@@ -1,7 +1,7 @@
 const axios = require('axios');
 const { Octokit } = require('@octokit/rest');
 
-const COMMIT_MESSAGE = 'Sync LeetCode submission';
+const COMMIT_MESSAGE = '[LeetCode Sync]';
```

Comment:
Given that this isn't backwards-compatible and is now customizable, maybe we can leave it out of this PR? 

If we did want to change the default, it would probably be good to still look for the old one during a transition period.